pcdsdevices/mirror.py

```diff
@@ -214,6 +215,53 @@ def removed(self):
         """Returns :keyword:`False`. Treats OffsetMirror as always inserted."""
         return False
 
+    def format_status_info(self, status_info):
+        """
+        Override status info handler to render the `OffsetMirror`.
+
+        Display `OffsetMirror` status info in the ipython terminal.
+
+        Parameters
+        ----------
+        status_info: dict
+            Nested dictionary. Each level has keys name, kind, and is_device.
+            If is_device is True, subdevice dictionaries may follow. Otherwise,
+            the only other key in the dictionary will be value.
+
+        Returns
+        -------
+        status: str
+            Formatted string with all relevant status information.
+        """
+        # happi metadata
+        try:
+            md = self.root.md
+        except AttributeError:
+            name = f'{self.prefix}'
+        else:
+            beamline = get_status_value(md, 'beamline')
+            functional_group = get_status_value(md, 'functional_group')
+            name = f'{self.prefix} ({beamline} {functional_group})'
+
+        p_position = get_status_value(status_info, 'pitch', 'position')
+        p_setpoint = get_status_value(status_info, 'pitch',
+                                      'setpoint', 'value')
+        p_units = get_status_value(status_info, 'pitch', 'setpoint',
+                                   'units')
+        p_p_volts = get_status_value(status_info, 'pitch', 'piezo_volts',
```

Comment:
Do we need the `piezo_volts` for the `OffsetMirror`?

Reply:
I think `piezo_volts` isn't needed here. Some background on the mechanics:

These pitch axes have two motors working together: a course stepper motor for moving long distances and a fine piezo motor for making small adjustments. So the piezo voltage is essentially wrapped up into the encoder readback for the pitch. It's a useful quantity but I think the scientists wanted to cut down on the number of things displayed here.

Reply:
thank you